src/Chapter1.hs

```diff
@@ -631,8 +634,13 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2::Int -> Int
+sumLast2 n = last n + prelast n
+  where
+    last::Int -> Int
+    last n = mod n 10
```

Comment:
`last` is the name of a standard function. Haskell allows name shadowing, that's why it works. Though, in practice, it's usually better to avoid name clashes with other functions cause it could be confusing 🙂

Also, I guess you could check what would happen on negative input here ;)